src/framework/auth/services/token/token.ts

```diff
@@ -19,13 +20,14 @@ export interface NbAuthRefreshableToken {
 
 export interface NbAuthTokenClass {
   NAME: string;
-  new (raw: any, ownerStrategyName: string): NbAuthToken;
+  new (raw: any, ownerStrategyName: string, createdAt?: Date): NbAuthToken;
```

Comment:
I believe createdAt shouldn't be optional here

Reply:
Right